pkg/ocmcontainer/ocmcontainer.go

```diff
@@ -125,26 +125,13 @@ func New(cmd *cobra.Command, args []string) (*ocmContainer, error) {
 	maps.Copy(c.Envs, backplaneConfig.Env)
 	c.Volumes = append(c.Volumes, backplaneConfig.Mounts...)
 
-	// Copy the ocm config into the container
```

Comment:
Moved this into the ocmConfig stuff, to match how the rest of these Envs and Mounts are created, for consistency